docs/installer/build-installer.rs

```diff
@@ -0,0 +1,30 @@
+use std::fs;
```

Comment:
curious why this is in the docs dir? (not necessarily opposed just not entirely clear why)

Reply:
Ah just threw it in here as it was deployed to gh-pages, would be fine to have it anywhere!